The unit tests on CI sometimes fail.

# Problem

The unit tests on CI sometimes fail.  

# Error messages

```shell
Run pip install -e .
Obtaining file:///home/runner/work/photo-backup/photo-backup
Installing collected packages: photo-backup
  Running setup.py develop for photo-backup
Successfully installed photo-backup-0.1.0
============================= test session starts ==============================
platform linux -- Python 3.7.10, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /home/runner/work/photo-backup/photo-backup
collected 28 items

test_exif.py .................                                           [ 60%]
test_photo.py .........F                                                 [ 96%]
test_upload.py .                                                         [100%]

=================================== FAILURES ===================================
_________________________ TestPhotosUpload.test_upload _________________________

self = <test_photo.TestPhotosUpload object at 0x7f24330cae50>

    def test_upload(self):
        photos = Photos(S3Uploader(consts.S3_BUCKET))
    
        photo1 = Photo("data/images/IMG_9235.jpeg")
        photos.append(photo1)
    
        photo2 = Photo("data/images/SOVK6553.jpeg")
        photos.append(photo2)
    
        photos.upload()
    
        s3 = boto3.client("s3")
        response = s3.list_objects_v2(Bucket=consts.S3_BUCKET, Prefix="2020")
        uploaded_filename = [
            content["Key"] for content in response["Contents"]
        ]
        expected = [
            "2020/08/20200813140708.jpeg",
            "2020/10/20201012131028.jpeg",
        ]
    
>       assert uploaded_filename == expected
E       AssertionError: assert ['2020/10/202...2131028.jpeg'] == ['2020/08/202...2131028.jpeg']
E         At index 0 diff: '2020/10/20201012131028.jpeg' != '2020/08/20200813140708.jpeg'
E         Right contains one more item: '2020/10/20201012131028.jpeg'
E         Use -v to get the full diff

test_photo.py:112: AssertionError
=========================== short test summary info ============================
FAILED test_photo.py::TestPhotosUpload::test_upload - AssertionError: assert ...
========================= 1 failed, 27 passed in 9.33s =========================
Error: Process completed with exit code 1.
```

--- a/tests/test_photo.py
+++ b/tests/test_photo.py
@@ -104,6 +104,8 @@ class TestPhotosUpload:
         uploaded_filename = [
             content["Key"] for content in response["Contents"]
         ]
+        uploaded_filename.sort()
+
         expected = [
             "2020/08/20200813140708.jpeg",
             "2020/10/20201012131028.jpeg",

